src/preact/index.js

```diff
@@ -8,15 +8,19 @@
  */
 
 // Importing directly from preact/dom is a hack to allow us to mimic the structure of react.
+
+/* eslint-disable */
 // @ts-ignore
-import * as preactDOM from 'preact/dom'
+import * as preactDOM from 'preact/dom';
 
```

Comment:
Instead of this, do `import {hydrate, render};` then type the exports

Reply:
What is the syntax for that? I don't think `export hydrate: TYPE` works

Reply:
Basically the same way you have it already:

```js
import {hydrate, render} from 'preact/dom';
/** @type {typeof import('preact').hydrate} */
export hydrate;
/** @type {typeof import('preact').render} */
export render;
```

Reply:
I don't believe that is valid syntax. Export requires either a declaration (`export const hydrate = ...`) or for it to be within a set of exports (`export {hydrate}`). 

I'll merge as-is, but if you figure out a way to make this work cleanly I'm all for it!

Reply:
Could we add `preact-dom.d.ts` with

```typescript
declare module 'preact/dom' {
  export const hydrate: typeof import('preact').hydrate;
  export const render: typeof import('preact').render;
}
```